# This expresses a transform between coordinate frames labeled by parent_id 
# and the frame_id field of the header.  

Header header
string parent_id # the frame id of the parent frame
Transform transform
